docs/usage-and-billing.mdx

```diff
@@ -112,15 +112,16 @@ Contract adjustment:
 
 ## Upgrade your plan
 
-To upgrade to the Semgrep Code **Team tier** through a **credit card**:
+To upgrade to the **Team tier** using a credit card:
 
-1. In the Settings page, select the **Usage & billing** tab.
-2. Select the number of developers to purchase licenses for.
-3. Fill in your payment details.
+1. Navigate to **Settings** > **Usage & billing**.
+2. Choose the product you want to upgrade. You can upgrade Semgrep Code, Semgrep Supply Chain, or both. Click **Upgrade**.
+3. Review the details of your order, and ensure that the number of licenses you're purchasing is accurate.
+4. Provide your payment details, and click **Subscribe**.
 
 ![Payment menu](/img/billing-and-pricing-payment.png)<br />
 
-To purchase licenses for Semgrep Supply Chain and Secrets, or to upgrade to the **Enterprise tier**, please [contact us](https://semgrep.dev/contact-us).
+To purchase licenses for Semgrep Secrets, or to upgrade to the **Enterprise tier**, please [contact us](https://semgrep.dev/contact-us).
```

Comment:
Your changes here make me think that the new section in licensing that you merged yesterday ([Claim a license](https://semgrep.dev/docs/licensing/#claim-a-license)) might be better suited to this document? WDYT? Not sure, but maybe "Pay for your plan" further down the Usage and billing doc can somehow be worked in as well.

Reply:
I think I'd prefer to keep 'em separate because there's no SCP screen if an AE provides you a license. What could be helpful is a link from the Usage & Billing page to the licensing page in case someone with that license wanders into this doc. Thoughts?

Reply:
Sure sounds good!

Reply:
Another quick thought: The licensing.md was originally for Semgrep's licenses, though, right? 
There's licensing as in, "Semgrep OSS is LGPL 2" and licensing as in "you bought a license to use commercial semgrep"

Reply:
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/81883788-1eba-461f-80ee-6511870008ac)

Reply:
Yeah, that's fair; what do you think about moving the Claim a license bit to the KB? (Unless you think there's a better spot for it?)

Reply:
Or it could be its own doc?

Reply:
Let me know what you think about where it's nested; I'm open to changing it!